src/vivarium_census_prl_synth_pop/results_processing/formatter.py

```diff
@@ -145,6 +145,11 @@ def format_columns(data: pd.DataFrame) -> pd.DataFrame:
     ) in COLUMN_FORMATTERS.items():
         if set(required_columns).issubset(set(data.columns)):
             data[output_column] = column_formatter(data[required_columns])
+    # Drop psimulate-specific columns that are not needed
+    drop_cols = ["input_draw", "scenario"]
```

Comment:
I think it's actually fine to leave these columns (new when doing psimulate now), but I drop them for consistency w/ the old results. My understanding is that we don't (and never will?) vary by scenario or draw for this model